packages/pwa/app/components/_app/index.jsx

```diff
@@ -168,6 +176,25 @@ const App = (props) => {
                             href={getAssetUrl('static/img/global/apple-touch-icon.png')}
                         />
                         <link rel="manifest" href={getAssetUrl('static/manifest.json')} />
+
+                        {/* Urls for all localized versions of this page (including current page)
+                        For more details on hrefLang, see https://developers.google.com/search/docs/advanced/crawling/localized-versions */}
+                        {SUPPORTED_LOCALES.map((locale) => (
+                            <link
+                                rel="alternate"
+                                hrefLang={locale.toLowerCase()}
+                                href={`${appOrigin}${buildUrlLocale(locale)}`}
+                                key={locale}
+                            />
+                        ))}
+                        {/* A general locale as fallback. For example: "en" if default locale is "en-GB" */}
+                        <link
+                            rel="alternate"
+                            hrefLang={defaultLocale.slice(0, 2)}
+                            href={`${appOrigin}${buildUrlLocale(defaultLocale)}`}
+                        />
+                        {/* A wider fallback for user locales that the app does not support */}
+                        <link rel="alternate" hrefLang="x-default" href={`${appOrigin}/`} />
```

Comment:
For all other user locales that the app does not support, Google suggests adding a link to the _homepage_.